centos安装扩展失败

编译最后一步 

`sudo make && make install
`
```
In file included from /usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:186:
/usr/local/src/php-ext-excel-export/library/third_party/minizip/crypt.h:35: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/usr/local/src/php-ext-excel-export/library/third_party/minizip/crypt.h:48: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/usr/local/src/php-ext-excel-export/library/third_party/minizip/crypt.h:65: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/usr/local/src/php-ext-excel-export/library/third_party/minizip/crypt.h:94: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c: In function ‘zipOpenNewFileInZip4_64’:
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:1245: error: ‘curfile64_info’ has no member named ‘crypt_header_size’
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:1251: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:1254: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:1255: error: ‘curfile64_info’ has no member named ‘crypt_header_size’
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c: In function ‘zip64FlushWriteBuffer’:
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:1375: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:1375: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c: In function ‘zipCloseFileInZipRaw64’:
/usr/local/src/php-ext-excel-export/library/third_party/minizip/zip.c:1610: error: ‘curfile64_info’ has no member named ‘crypt_header_size’
make: *** [library/third_party/minizip/zip.lo] Error 1
```

环境
PHP 7.2.13 (cli) (built: Dec  8 2018 12:27:01) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.7, Copyright (c) 1999-2018, by Zend Technologies

rinting-4.0-noarch
Distributor ID:	CentOS
Description:	CentOS release 6.7 (Final)
Release:	6.7
Codename:	Final

> @needrunning
> 
> Has your problem been solved?

now I have two problems 

one：    in order to install pxlswriter  via pecl install pxlswriter,  Do you need to reinstall PHP？

two： I have  runed     pecl install pxlswriter  in my server ，
As shown below

lude/php -I/usr/include/php/main -I/usr/include/php/TSRM -I/usr/include/php/Zend -I/usr/include/php/ext -I/usr/include/php/ext/date/lib -I/tmp/pear/install/xlswriter/library/include -I/tmp/pear/install/xlswriter -I/tmp/pear/install/xlswriter/include -DHAVE_CONFIG_H -g -O2 -c /tmp/pear/install/xlswriter/library/third_party/minizip/zip.c  -fPIC -DPIC -o library/third_party/minizip/.libs/zip.o
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:160: error: expected ‘:’, ‘,’, ‘;’, ‘}’ or ‘__attribute__’ before ‘*’ token
In file included from /tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:186:
/tmp/pear/install/xlswriter/library/third_party/minizip/crypt.h:35: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/tmp/pear/install/xlswriter/library/third_party/minizip/crypt.h:48: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/tmp/pear/install/xlswriter/library/third_party/minizip/crypt.h:65: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/tmp/pear/install/xlswriter/library/third_party/minizip/crypt.h:94: error: expected ‘;’, ‘,’ or ‘)’ before ‘*’ token
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c: In function ‘zipOpenNewFileInZip4_64’:
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:1245: error: ‘curfile64_info’ has no member named ‘crypt_header_size’
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:1251: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:1254: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:1255: error: ‘curfile64_info’ has no member named ‘crypt_header_size’
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c: In function ‘zip64FlushWriteBuffer’:
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:1375: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:1375: error: ‘curfile64_info’ has no member named ‘pcrc_32_tab’
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c: In function ‘zipCloseFileInZipRaw64’:
/tmp/pear/install/xlswriter/library/third_party/minizip/zip.c:1610: error: ‘curfile64_info’ has no member named ‘crypt_header_size’
make: *** [library/third_party/minizip/zip.lo] Error 1

--- a/config.m4
+++ b/config.m4
@@ -103,6 +103,8 @@ if test "$PHP_XLSWRITER" != "no"; then
         if test `echo $XLSXWRITER_VERSION | $SED -e 's/[[^0-9]]/ /g' | $AWK '{print $1*10000 + $2*100 + $3}'` -ge 807; then
             AC_DEFINE(HAVE_WORKBOOK_ADD_VBA_PROJECT, 1, [ workbook_add_vba_project available in 0.8.7 ])
         fi
+        dnl see library/CMakeLists.txt
+        LIBOPT="-DNOCRYPT -DNOUNCRYPT"
     fi
 
     if test -z "$PHP_DEBUG"; then
@@ -111,7 +113,7 @@ if test "$PHP_XLSWRITER" != "no"; then
 
     PHP_SUBST(XLSWRITER_SHARED_LIBADD)
 
-    PHP_NEW_EXTENSION(xlswriter, $xls_writer_sources, $ext_shared,, -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1)
+    PHP_NEW_EXTENSION(xlswriter, $xls_writer_sources, $ext_shared,, -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 $LIBOPT)
 
     PHP_ADD_INCLUDE([$srcdir])
     PHP_ADD_INCLUDE([$srcdir/include])

